src/components/Text/text/text.scss

```diff
@@ -0,0 +1,73 @@
+@use '../../variables' as variables;
+@use '../../../../styles/mixins' as mixins;
```

Comment:
By default its already in these namespaces (ns = filename)

Reply:
removed `as variables` and `as mixins;`;